Std/Data/List/Basic.lean

```diff
@@ -823,6 +823,36 @@ inductive Forall₂ (R : α → β → Prop) : List α → List β → Prop
 
 attribute [simp] Forall₂.nil
 
+theorem forall₂_cons {R : α → β → Prop} {a b as bs} :
```

Comment:
```suggestion
@[simp]
theorem forall₂_cons {R : α → β → Prop} {a b as bs} :
```
to match the version in Mathlib.

Perhaps it makes sense to upstream the entire [file](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/List/Forall2.html#List.forall%E2%82%82_cons) in a follow-up PR, to avoid accidentally going out of sync.